tests/unit/report-command/language-files.spec.ts

```diff
@@ -33,8 +33,14 @@ describe('file: report-command/language-files', () => {
 
     it('Create fails if language files are not valid JSON', () => {
```

Comment:
btw, it doesn't looks like this is fails.

This is incorrect description or am i missing something?

Reply:
Yeah, that's going to my first bit of business— is getting all the tests written for the new use cases people have brought up for the tool and making sure that the existing tests still all work and test the right conditions.